helm/alfresco-content-services/values.yaml

```diff
@@ -253,6 +253,7 @@ share:
 persistence:
   existingClaim: "alfresco-volume-claim"
   repository:
+    enabled: true
```

Comment:
I think this switch needs to be documented together with `s3connector.enabled` describing what they do.
Please take a look at the table of possible values for the Helm chart here:
https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services#configuration

Reply:
done